EIPS/eip-67.md

```diff
@@ -0,0 +1,80 @@
+---
+eip: 67
+title: URI Scheme with Metadata, Value and Bytecode
+description: Format for encoding transactions into a URI
+author: Alex Van de Sande (@alexvandesande)
+discussions-to: https://github.com/ethereum/EIPs/issues/67
+status: Withdrawn
+type: Standards Track
+category: ERC
+created: 2016-02-17
+withdrawal-reason: Superseded by EIP-831
```

Comment:
```suggestion
withdrawal-reason: Superseded by [EIP-831](./eip-831.md)
```

Reply:
Huh, is markdown supported in the preamble? I've always assumed it wasn't!

Reply:
Maybe it isn't. This is fine.